public/static/admin/config.yml

```diff
@@ -18,9 +18,11 @@ collections:
       fields:
         - {label: "Heading", name: "heading", widget: "string"}
         - {label: "Intro", name: "intro", widget: "markdown"}
-        - label: "News"
+        - label: "Stories"
+          label_singular: "Story"
```

Comment:
This is a bonus change to correct the labels for this section in the CMS.

![Screen Shot 2020-10-25 at 5 16 13 PM](https://user-images.githubusercontent.com/7942714/97122816-d9a93900-16e5-11eb-860a-271ebe033ef3.png)